tests/letstest/targets.yaml

```diff
@@ -13,40 +24,25 @@ targets:
     type: ubuntu
     virt: hvm
     user: ubuntu
-  - ami: ami-09677e0a6b14905b0
-    name: ubuntu16.04LTS
-    type: ubuntu
-    virt: hvm
-    user: ubuntu
   #-----------------------------------------------------------------------------
   # Debian
   - ami: ami-01db78123b2b99496
     name: debian10
     type: ubuntu
     virt: hvm
     user: admin
-  - ami: ami-003f19e0e687de1cd
-    name: debian9
-    type: ubuntu
-    virt: hvm
-    user: admin
-  - ami: ami-0ed54dd1b25657636
-    name: debian9_arm64
-    type: ubuntu
-    virt: hvm
-    user: admin
-    machine_type: a1.medium
-    # userdata: |
-    #   #cloud-init
-    #   runcmd:
-    #     - [ apt-get, install, -y, curl ]
   #-----------------------------------------------------------------------------
   # Other Redhat Distros
   - ami: ami-0916c408cb02e310b
     name: RHEL7
     type: centos
     virt: hvm
     user: ec2-user
+    userdata: |
+      #cloud-config
+      runcmd:
+        - yum-config-manager --enable rhui-REGION-rhel-server-extras rhui-REGION-rhel-server-optional
+        - yum install -y python3-devel.x86_64
```

Comment:
nit: Is this line needed with `BootstrapRpmPython3` installing `python3-devel`? My understanding is the two package names are equivalent except this one hard codes the architecture while the other will just use the architecture of the machine its running on which should also be x86_64.

Reply:
It does seem to be required, yes. It couldn't find `python3-devel`, which is why I added this here. Various q&as I found all do seem to indicate you specifically need the architecture on this machine, as well.

Reply:
"elsewhere" being a few lines down, here: https://github.com/certbot/certbot/pull/8207/files#diff-776152681de94287606432a9922f03f8L81

Reply:
It couldn't find `python3-devel` even after you enabled the necessary repos? That surprises me but trying to install the package twice doesn't hurt anything.

Reply:
it's possibly that I didn't test that iteration! there were many things I tried. I'll double-check.

Reply:
you're right, we don't need to specify the arch

Reply:
ah, I've just remembered -- it's because `python3-devel` is neither necessary nor available on RHEL/centos8.

Reply:
`yum install python3-devel` seems to work on CentOS/RHEL 8, but `yum list python3-devel` does not.

I think we can fix the `list` command by changing the command to `if ! $TOOL list 'python3*-devel' >/dev/null 2>&1; then`.

The problem seems to be that `python3-devel` is an alias for `python36-devel` with the install subcommand on CentOS/RHEL 8, but not the `list` subcommand.

Reply:
oh, I like that better than how I have it.